cms/djangoapps/contentstore/views/course.py

```diff
@@ -1422,6 +1453,18 @@ def group_configurations_detail_handler(request, course_key_string, group_config
             return JsonResponse(status=204)
 
 
+def has_course_enabled_content_experiments(course):
+    """
+    Returns True if Studio should show the "Experiment Group
```

Comment:
I've never seen a method starting with ```are_``` but I see that there are two of them in edx-platform! It is tricky because it is a singular feature (content experiments) that sounds like it is plural. I'm okay with ``are_content_experiments_enabled``.